core-services/prow/02_config/_boskos.yaml

```diff
@@ -1261,6 +1261,10 @@ resources:
 - names:
   - us-east-1--aws-sustaining-autorelease-412-quota-slice-0
   - us-east-1--aws-sustaining-autorelease-412-quota-slice-1
+  - us-east-1--aws-sustaining-autorelease-412-quota-slice-2
+  - us-east-1--aws-sustaining-autorelease-412-quota-slice-3
+  - us-east-1--aws-sustaining-autorelease-412-quota-slice-4
+  - us-east-1--aws-sustaining-autorelease-412-quota-slice-5
```

Comment:
slice-0 to slice-5 are 6 quota slice, which mismatch with the expected 5 quota slice.

Reply:
@liangxia Thanks for pointing out. Please help to review.